### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by Batman9698**

### CODE REVIEW

**Comment by eric-wieser** [comment]
"ABI" means "application binary interface". Can you explain why you think that was not what was intended here?

File: numpy/__init__.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def __dir__():
     def _sanity_check():
         """
         Quick sanity checks for common bugs caused by environment.
-        There are some cases e.g. with wrong BLAS ABI that cause wrong
+        There are some cases e.g. with wrong BLAS API that cause wrong
```
Comment (bashtage) [inline numpy/__init__.py:335]: I think the problem is that a BLAS can have the wrong Application Binary Interface, meaning it was built with the wrong compiler or toolchain, for the platform it is running on.  Wrong API means wrong functions exposed which is pretty rare.

Comment (bashtage) [reply numpy/__init__.py:335]: https://en.wikipedia.org/wiki/Application_binary_interface

**Comment by seberg** [comment]
Agreed, thanks for opening a PR, but "API" would not be correct here.  If the wrong symbols where exposed, I don't think we would get this far, the library would fail to load much earlier.

**Pull request closed without merging**